### DISCUSSION TITLE
Multi table filter ideas?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#8894] [General - Components / Directives / etc] [answered]
Is there an easy way to adapt the search filter for a table to apply to multiple, separate tables on a page? Or should I just write a custom component at the top of the page that filters the data of each of the separate tables when activated?

**Comment by odranoelBR** [comment, marked as answer]
Hi @ssuess 

If I understand, simple props can do the job
Check here: https://codesandbox.io/s/white-hooks-6dxfg?file=/src/pages/Index.vue

**Reply by ssuess** [reply]
Thanks that seems way simpler than I thought it would be! I will try it out.

**Reply by ssuess** [reply]
it worked great, but one thing I can't figure out is how to get the number of results from each table after filtering, any ideas? For example, I have a number at the top of my page that counts the total number of results from my data api call (that are then spread across multiple tables) and I would like to have that number change when the filter has been applied, but can't figure out how to grab the number of filtered results for each table to update the total.

**Reply by odranoelBR** [reply]
Awesome @ssuess !

If you use server side pagination: https://quasar.dev/vue-components/table#server-side-pagination-filter-and-sorting, 
the @request event from QTable will give you the total.

**Reply by ssuess** [reply]
alas, this is all client side manipulation after getting the data from the api...

**Reply by odranoelBR** [reply]
For client side try this: https://codesandbox.io/s/white-hooks-6dxfg?file=/src/pages/Index.vue

If you can, show a picture o/

**Reply by ssuess** [reply]
Perfect! Thank you so much!
